library/public/image.h

```diff
@@ -45,13 +46,13 @@ class F3D_EXPORT image
   };
 
   /**
-   * Create an image from file, the following formats are supported:
+   * Create an image from provided file path, used as is, the following formats are supported:
```

Comment:
```suggestion
   * Read an image from provided file path, used as is, the following formats are supported:
```
"Create" kind of implies making a new/blank one to me? "Read [from]" (or "Load [from]") matches better with the "Save [to]" used later on.

Although I guess it depends whether "image" is supposed to be read as a regular noun or a c++ type because a constructor does indeed _create_ an instance
```suggestion
   * Create an `image` from provided file path, used as is, the following formats are supported:
```

Reply:
reworded